pom.xml

```diff
@@ -178,6 +178,7 @@ alternative support for serializing POJOs as XML and deserializing XML as pojos.
           <configuration>
             <excludes>
               <exclude>com/fasterxml/jackson/dataformat/xml/failing/*.java</exclude>
```

Comment:
Shouldn't we remove above line as redundant?

Reply:
Right, thanks!